examples/OCPS.ipynb

```diff
@@ -0,0 +1,341 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    ocps = salobj.Remote(domain, "OCPS")
</pre><p>I think you should add an index.</p><p>I might also be worth indicating that the index should be camera specific? e.g. ComCam is always index 1</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/47/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
I had never really decided whether we would have one OCPS for the entire system (hence no need for an index) or a separate OCPS instance for each camera (as implied by the configurations).  I'm tending towards the first rather than the second.  What do you think?

Reply:
I don't really see why we would need it to be indexed.... what scared me this time was when i saw the config and worried that it didn't support indexes :) 
Does having it indexed help guard against bugs? Meaning could a bug related to AuxTel OCPS usage negatively impact another camera?